### PR TITLE
updated the afk message and changed the snipe desc

### PR DESCRIPTION
**Pull request by ConnorDoesDev**

### CODE REVIEW

File: cogs/misc.py [hunk 1]
```diff
@@ -352,17 +352,17 @@ async def setafk(self, ctx, *, note: commands.clean_content = "I'm currently AFK
         if check is not None:
             await self.bot.db.execute("UPDATE afk SET message = $1 WHERE user_id = $2 AND guild_id = $3", note, ctx.author.id, ctx.guild.id)
             self.bot.afk[f"{str(ctx.guild.id)}, {str(ctx.author.id)}"]['note'] = note
-            await ctx.send(_("{0} **Changed your AFK state to -** {1}").format(
+            await ctx.send(_("{0} **Changed your AFK state to**: {1}").format(
```
Comment (TheMoksej) [inline cogs/misc.py]: this should be before the asterisks

**Review by TheMoksej — CHANGES_REQUESTED** [review]
Everything looks good, except the `:`. It's outside the asterisks

File: cogs/misc.py (hunk 1 above)
Comment (ConnorDoesDev) [reply cogs/misc.py]: if I understand how GitHub works, I fixed that

File: cogs/misc.py [hunk 2]
```diff
@@ -352,17 +352,17 @@ async def setafk(self, ctx, *, note: commands.clean_content = "I'm currently AFK
...
         elif check is None:
             await self.bot.db.execute("INSERT INTO afk(user_id, guild_id, message, time) VALUES($1, $2, $3, $4)", ctx.author.id, ctx.guild.id, note, datetime.now())
             self.bot.afk[f"{str(ctx.guild.id)}, {str(ctx.author.id)}"] = {'note': note, 'time': datetime.now()}
-            await ctx.send(_("{0} ** Set your AFK state to -** {1}").format(
+            await ctx.send(_("{0} ** Set your AFK state to:** {1}").format(
                 self.bot.settings['emojis']['misc']['white-mark'], escape_markdown(note, as_needed=False)
             ))
 
-    @commands.command(brief='Snipe the latest deleted message')
+    @commands.command(brief='Snipe a deleted message')
```
Comment (DuckMasterAl) [inline cogs/misc.py]: You're not sniping any deleted message; you're sniping the latest message in the channel.
This means the description should be "See the latest deleted message in a channel."

**Review by DuckMasterAl — CHANGES_REQUESTED** [review]

**Review by DuckMasterAl — APPROVED** [review]

**Pull request merged**